cves/kernel/CVE-2015-8787.yml

```diff
@@ -448,7 +480,11 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    A NULL check was removed, 
+    'if (indev != NULL) {' in favor of 'if (indev && indev->ifa_list) {'
+    which allowed for a NULL value to be accepted. A similar issue occurred in
+    2003 but the 2015 commit was not properly reviewed.
```

Comment:
Maybe expand on this a little bit more. Was this a planning issue? A mistake? And how was the 2015 commit not properly reviewed? Remember those reading this won't know all of the context you know from your research.